querydsl-core/src/main/java/com/mysema/query/types/OperatorImpl.java

```diff
@@ -81,4 +58,18 @@ public String toString() {
     public int hashCode() {
         return hashCode;
     }
+
+    private Object readResolve() throws ObjectStreamException {
+        try {
+            String[] names = id.split("#");
+            Field opField = Class.forName(names[0]).getDeclaredField(names[1]);
```

Comment:
Consider using `getField()` instead, it's implementation in OpenJDK 8 seems more performant.
Also the semantics of that method apply better in this case, since it is a public field.